src/scripts/main.js

```diff
@@ -1,3 +1,43 @@
 'use strict';
 
-// write your code here
+document.querySelector('.logo').addEventListener('click', () => {
+  callPromise1();
+  callPromise2();
```

Comment:
you don't need to call the second promise on click, just call it immediately

Reply:
Done

Reply:
task description is so unclear